src/common/utils/validators.py

```diff
@@ -146,7 +146,28 @@ def _validate_entity(
                 attributeDefinition, entity[attributeDefinition.name], f"{key}.{attributeDefinition.name}"
             )
         elif attributeDefinition.attribute_type == "any" and attributeDefinition.name == "default":
-            continue
+            default_attribute_definition = BlueprintAttribute(
+                name=attributeDefinition.name,
+                attribute_type=entity["attributeType"],
+                dimensions=entity.get("dimensions", None),
+            )
+            if default_attribute_definition.is_array:
+                _validate_list(
+                    entity["default"],
+                    default_attribute_definition,
+                    get_blueprint,
+                    f"{key}.{default_attribute_definition.name}",
+                    len(default_attribute_definition.dimensions.dimensions),
+                    implementation_mode,
```

Comment:
We want to check that the default value is valid in itself (ie it implements its own type) and that it is either of same type as defined in the attribute definition or extending from it
```suggestion
                    "extend",
```